web/src/app/components/bar-chart/bar-chart.component.ts

```diff
@@ -629,4 +703,91 @@ export class BarChartComponent implements AfterViewInit {
         .style('font', '11px sans-serif')
         .style('fill', BarChartComponent.COLOR_LIGHT_GRAY);
   }
+
+  /**
+   * The function that adds tag content, which is the time duration for the
```

Comment:
I'd say "This function adds ....." instead of "The function that adds...". The former sounds more natural to me. 
Same for the function comment in other places.